### ISSUE TITLE
Documentation issue: Cyclomatic Complexity Rule

### ISSUE THREAD

**Issue by danielsitnik** [palantir/tslint#2822]
The documentation for the [Cyclomatic Complexity Rule](https://palantir.github.io/tslint/rules/cyclomatic-complexity/) states that:

> A starting value of **20** is assigned and this value is then incremented for every statement which can branch the control flow within the function

From my experience here, the starting value is actually 1.
You can see from the bug report below that I'm using a threshold of 3 in the configuration, and tslint is correctly complaining that my function has a cyclomatic complexity of 4. So the default starting value is definitely not 20.

### Bug Report

- __TSLint version__: 5.3.2
- __TypeScript version__: 2.1.0
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
export class Test {

    test(term: string): boolean {
        if (term) {
            if (term.length > 0) {
                if (term === 'test') {
                    console.log('test');
                }
            }
        }

        return true;
    }
}
```

with `tslint.json` configuration:

```json
"rules": {
    "cyclomatic-complexity": [true, 3]
}
```

#### Actual behavior
severity: 'Error'
message: 'The function test has a cyclomatic complexity of 4 which is higher than the threshold of 3 (cyclomatic-complexity)'
at: '20,5'
source: 'tslint'

**Comment by ajafff** [comment]
You're right, it's not the starting value. 20 is used as default if not threshold is provided by the config
